specs/lib/cacheOptimizeIndicesSpec.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var cacheOptimizeIndices = require('../../lib/cacheOptimizeIndices');
+var addDefaults = require('../../lib/addDefaults');
+var readGltf = require('../../lib/readGltf');
+var readAccessor = require('../../lib/readAccessor');
+
+var gltfPath = './specs/data/cacheOptimizeIndices/chinesedragon.gltf';
```

Comment:
The cache optimization seems to have no effect on the usual box we use for testing -- I assumed this was reasonable since there are so few indices. Let me see if I can figure out how to reorder the indices such that they will be moved in a known manner with the cache optimization so I can avoid using the dragon model.